drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModel.java

```diff
@@ -266,6 +267,7 @@ public void deleteColumn(int columnIndex) {
         final GridColumn<?> toDelete = getColumns().get(columnIndex);
         deleteColumn(toDelete);
         simulation.removeFactMappingByIndex(columnIndex);
+        synchronizeFactMappingsWidths();
```

Comment:
@yesamer 
I think that's to be removed. `synchronizeFactMappingsWidths` put the current columns widht inside factmappings, but we decided it should happen **only** when the user saves the scenario - otherwise rely on the "status" implementation to manage them

Reply:
Done!